cegs_portal/search/templates/search/v1/source_reg_effects.html

```diff
@@ -3,60 +3,12 @@
-            <th class="chrom-dark-band">Direction</th>
-            <th class="chrom-right-centromere chrom-light-band">Significance<br />(p-value)</th>
-            <th class="chrom-left-centromere chrom-light-band">Experiment</th>
-            <th class="chrom-last-end-cap chrom-dark-band">Target Gene</th>
-        </tr>
-        {% for regeffect in regeffects %}
-            {% for re_target in regeffect.targets.all %}
-            <tr>
-                {% if forloop.first %}
-                <td rowspan="{% firstof regeffect.targets.all|length 1 %}" scope="rowgroup"><a href="{% url 'search:reg_effect' regeffect.accession_id %}">{{ regeffect.accession_id }}</a></td>
-                <td rowspan="{% firstof regeffect.targets.all|length 1 %}" scope="rowgroup">{{ regeffect.effect_size }}</td>
-                <td rowspan="{% firstof regeffect.targets.all|length 1 %}" scope="rowgroup">{{ regeffect.direction }}</td>
-                <td rowspan="{% firstof regeffect.targets.all|length 1 %}" scope="rowgroup">{{ regeffect.significance }}</td>
-                <td rowspan="{% firstof regeffect.targets.all|length 1 %}" scope="rowgroup"><a href="{% url 'search:experiment' regeffect.experiment.accession_id %}">{{ regeffect.experiment.name }}</a></td>
-                {% endif %}
-            <td><a href="{% url 'search:dna_features' 'accession' re_target.accession_id %}">{{ re_target.name }}</a></td>
-            </tr>
-            {% empty %}
-                <tr data-href="{% url 'search:reg_effect' regeffect.accession_id %}">
-                    <td><a href="{% url 'search:reg_effect' regeffect.accession_id %}">{{ regeffect.accession_id }}</a></td>
-                    <td>{{ regeffect.effect_size }}</td>
-                    <td>{{ regeffect.direction }}</td>
-                    <td>{{ regeffect.significance }}</td>
-                    <td><a href="{% url 'search:experiment' regeffect.experiment.accession_id %}">{{ regeffect.experiment.name }}</a></td>
-                    <td>-</td>
-                </tr>
-            {% endfor %}
-        {% endfor %}
-    </table>
 
-    <div class="pagination">
-        <span class="step-links">
-            {% if regeffects.has_previous %}
-                <a href="?page=1">&laquo; first</a>
-                <a href="?page={{ regeffects.previous_page_number }}">previous</a>
-            {% endif %}
-
-            <span class="current">
-                Page {{ regeffects.number }} of {{ regeffects.paginator.num_pages }}
-            </span>
-
-            {% if regeffects.has_next %}
-                <a href="?page={{ regeffects.next_page_number }}">next</a>
-                <a href="?page={{ regeffects.paginator.num_pages }}">last &raquo;</a>
-            {% endif %}
-        </span>
-    </div>
+{% if feature %}
+<div class="container min-w-full" id="reg-table" hx-get="{% url 'search:source_effects' feature.accession_id %}" hx-trigger="load" >
```

Comment:
This doesn't need to be triggered on load, that will cause a "double load" -- the regular page load has the regeffect data and then the hx-trigger will get the same data again, I think.